### PR TITLE
SFR 2238/run gha on local

### PR DESCRIPTION
**Pull request by ayan1229**
[Jira Ticket](https://newyorkpubliclibrary.atlassian.net/browse/SFR-2238)

### This PR does the following:
- updates the Playwright.yml file to trigger GHA test on every pull request on localhost server

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "extract-loader": "^5.1.0",
     "file-loader": "^6.2.0",
     "focus-trap-react": "^10.0.0",
-    "newrelic": "12.5.0",
+    "newrelic": "^12.5.0",
```
Comment (jackiequach) [inline package.json:28]: This should not need to be changed.

**Review by jackiequach — COMMENTED** [review]
This would still be using the drb qa since it was not changed in the `cucumber.json` file

File: .github/workflows/Playwright.yml [hunk 2]
```diff
@@ -19,6 +19,22 @@ jobs:
...
+          docker-compose up -d
+        working-directory: ./sfr-bookfinder-front-end
+
+      - name: Update /etc/hosts
+        run: |
+          echo "127.0.0.1 local.nypl.org" | sudo tee -a /etc/hosts
+
+      - name: Wait for the application to start
+        run: |
+          sleep 30 # Ensure the app has enough time to start up
```
Comment (kylevillegas93) [inline .github/workflows/Playwright.yml:36]: nit: feel free to remove the comment. I think the name of the task is descriptive

File: cucumber.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "default": {
     "import": ["playwright/tests/**/*.ts", "playwright/support/**/*.ts"],
     "worldParameters": {
-      "appUrl": "https://drb-qa.nypl.org",
+      "appUrl": "http://localhost:3000",
```
Comment (jackiequach) [inline cucumber.json:5]: If login is tested you may want to use local.nypl.org

**Review by jackiequach — APPROVED** [review]

**Review by kylevillegas93 — APPROVED** [review]

**Pull request merged**